docs/modules/indexes/document_loaders/examples/example_data/notebook.ipynb

```diff
@@ -25,7 +25,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "loader = NotebookLoader(\"example_data/notebook.ipynb\")"
+    "loader = NotebookLoader(\"../../../example_data/notebook.ipynb\")"
```

Comment:
this shouldnt really need to change?

Reply:
No. We don't need to change test data. Thanks.

> hmmm in theory i like this, but the distinction seems a bit blurred. for example, why is Google Drive a formatter and not a knowledge document loader

I'll add more description to this. 
The idea is "knowledge loader" works with storage that we do not control. Something that can be used as a "tool" (in terms of LangChain). That can be accessed with queries. Something that can be considered as a source of "external" knowledge. We can allow LLM to make queries and get information or we can download documents and use them in a more controllable way.
"Formatters" can be as easy as transformers for CSV, SQL, etc. But they also can be cloud services or app stores. They can be hosted out of our control but the information inside is under our control.